purchases/src/main/kotlin/com/revenuecat/purchases/listenerConversions.kt

```diff
@@ -410,3 +412,21 @@ fun Purchases.getNonSubscriptionSkusWith(
 ) {
     getNonSubscriptionSkus(skus, getSkusResponseListener(onReceiveSkus, onError))
 }
+
+/**
+ * Check if billing is supported for the current Play user (meaning IN-APP purchases are supported)
+ * and optionally, whether a list of specified feature types are supported. This method is asynchronous
+ * since it requires a connected BillingClient.
+ * @param context A context object that will be used to connect to the billing client
+ * @param feature A list of feature types to check for support. Feature types must be one of [BillingFeature]
+ *                 By default, is an empty list and no specific feature support will be checked.
+ * @param callback Callback that will be notified when the check is complete.
+ */
+@Suppress("unused")
+fun Purchases.Companion.canMakePaymentsWith(
+    context: Context,
+    features: List<BillingFeature> = listOf(),
+    onComplete: (Boolean) -> Unit
```

Comment:
Being `Callback<Boolean>` a single abstract method interface, which has automatic SAM conversion, I don't think we really need to have a specific conversion for Kotlin since it already looks nice when calling the regular `canMakePayments`. What do you think?

Reply:
agreed! for some reason didn't think Callback could be references from other repos but it definitely can. i'll refactor